### ISSUE TITLE
Feature request: Show the description of an exception

### ISSUE THREAD

**Issue by insolor** [Dman95/SASM#24]
Currently SASM doesn't show the exception message.
For example, I write such code:

``` assembly
format ELF

section '.text' executable
public _main
_main:
    mov ebp, esp; for correct debugging
    ;write your code here
    mov edx, 0
    mov eax, 100
    mov ecx, 0
    div ecx
    xor eax, eax
    ret
```

Then press "Debug" and "Continue". As expected, the debugger stops at the `div ecx` instruction because of "Integer division by zero" exception (as OllyDbg describes it), but SASM just stops and doesn't show any message. This isn't critical for this particular case, but a message with the description of an exception could be useful in more complicated situations.

Windows 8.1, SASM 3.1.2, assemble with the Fasm shipped with SASM

**UPD.** Just tested with NASM - it doesn't show the exception message neither.

**Comment by Dman95** [comment]
Feature has been implemented in v3.1.3

**Comment by insolor** [comment]
Оперативно )
Thanks!
